design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -0,0 +1,38 @@
+type ComparableItem {
+    productId: ID!
+    name: String!
+    priceRange: ProductPriceRange!
+    url: ProductUrl
+    images: [ProductImage]
+    values: [ProductAttribute]
+}
+
+type ComparableIndicator {
+    code: String!
+    title: String!
+}
+
+type CompareList {
+    items: [ComparableItem]
+    indicators: [ComparableIndicator]
+}
+
+type Customer{
+    compareList: CompareList
+}
+
+input CompareListInput {
+    id: ID!
+    items: [ID!]
```

Comment:
Why not product SKU? Everywhere else in GraphQL we operate based on product SKU.

Reply:
SKU is not a unique guaranteed identifier for the catalog.
Along with this product exposes ID as a part of product type.